lm_eval/tasks/__init__.py

```diff
@@ -60,8 +60,7 @@ def check_prompt_config(config: dict[str, str]) -> List[dict[str, str]]:
                                 prompt_variation,
                             ]
                         )
-                    },
-                    **{"output_type": "greedy_until"},
+                    }
                 }
```

Comment:
Will need to figure out how to enable `greedy_until` and `multiple_choice` modes and how to handle them separately. For now, let's revert this back.